.travis.yml

```diff
@@ -12,13 +14,24 @@ matrix:
   fast_finish: true
   include:
     - python: "pypy"
+      dist: trusty
     - python: "pypy3"
-    - python: '3.7-dev'
+      dist: trusty
+    - python: '3.7'
+    - python: '2.7'
     - python: '2.7'
+      dist: trusty
     - python:  "2.7_with_system_site_packages" # For PyQt4
+    - python:  "2.7_with_system_site_packages" # For PyQt4
+      dist: trusty
+    - python: '3.6'
     - python: '3.6'
+      dist: trusty
     - python: '3.5'
+    - python: '3.5'
+      dist: trusty
     - python: '3.4'
+      dist: trusty
```

Comment:
You've missed 3.4 on Xenial.

Reply:
Nevermind, I see now.

Reply:
PyPy2/3 and 3.4 only support Trusty on Travis.

Here's how they look: https://travis-ci.org/hugovk/Pillow/builds/397828420